From 0012e747d7819631343aad519d0532b8c7775a10 Mon Sep 17 00:00:00 2001
From: John McFarlane <john.mcfarlane@rockfloat.com>
Date: Wed, 10 Feb 2010 15:31:15 -0800
Subject: [PATCH 2/2] [2.5 BACKPORT] Allow setup with python-2.5

---
 setup.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index f2a8963..caae930 100644
--- a/setup.py
+++ b/setup.py
@@ -26,8 +26,8 @@ from chula import error
 
 # Check for dependencies
 if 'install' in sys.argv:
-    if sys.version_info < (2, 6):
-        raise error.MissingDependencyError('Python-2.6 or higher')
+    if sys.version_info < (2, 5):
+        raise error.MissingDependencyError('Python-2.5 or higher')
 
 # Attributes
 AUTHOR = 'John McFarlane'
-- 
1.6.4.4

