diff --git a/setup.py b/setup.py
index d65a978..75b75e5 100644
--- a/setup.py
+++ b/setup.py
@@ -26,8 +26,8 @@ from chula import error
 
 # Check for dependencies
 if 'install' in sys.argv:
-    if sys.version_info < (2, 6):
-        raise error.MissingDependencyError('Python-2.6 or higher')
+    if sys.version_info < (2, 5):
+        raise error.MissingDependencyError('Python-2.5 or higher')
 
 def find_packages():
     for root, dirs, files in os.walk('chula'):
